<launch>
  <arg name="uri" default="radio://0/80/2M/E7E7E7E7A0" />
  <arg name="frame" default="base_link" />
  <arg name="joy_dev" default="/dev/input/js0" />
  <arg name="x" default="0.0" />
  <arg name="y" default="0.0" />
  <arg name="z" default="0.0" />

  <param name="robot_description" command="$(find xacro)/xacro.py $(find crazyflie_description)/urdf/crazyflie2.urdf.xacro" />
  <node name="rviz" pkg="rviz" type="rviz"
        args="-d $(find bitcraze_lps_estimator)/data/rvizconfig_with_goal.rviz"/>

  <node name="joy" pkg="joy" type="joy_node" output="screen">
    <param name="dev" value="$(arg joy_dev)" />
  </node>

  <group ns="crazyflie">
    <remap from="joy" to="/joy"/>
    <include file="$(find iconslab)/launch/shape_crazyflie.launch">
      <arg name="ns" value="crazyflie"/>
      <arg name="uri" value="$(arg uri)"/>
      <arg name="shape" value="figure8"/>
      <arg name="hold" value="True"/>
      <arg name="theta" value="0"/>
    </include>
  </group>


  <node pkg="tf" type="static_transform_publisher" name="link1_broadcaster"
        args="1 0 0 0 0 0 1 world lps 100" />

  <include file="$(find crazyflie_driver)/launch/crazyflie_server.launch" />
</launch>
